src/main/java/performance/LocustOperations.java

```diff
@@ -98,15 +99,41 @@ private void clearValues() {
      * This method execute the sequence needed for run the test
      */
     public void executePerformanceTask(DataTable testData) throws Exception {
-        this.setTestData(testData);
-        TimeUnit.SECONDS.sleep(5);
-        this.executeMaster();
-        TimeUnit.SECONDS.sleep(10);
-        this.setUpSlave();
-        this.executeTask(testData);
-        TimeUnit.MINUTES.sleep(this.testTime);
-        this.locust.stop();
-        this.clearValues();
```

Comment:
So, finally there was no need for something in Linux?

Reply:
Yes, you can find it in the line 120 in the green section...